File: tempo-rise-webapp/src/app/core/models/avis.ts
import { Product } from "./product";
import { User } from "./user";

export interface Avis {
    id: number;
    note: number;
    commentaire: string;
    datePublication: string;
    utilisateurId: number;
    produitId: number;
    utilisateur:User 
    produit:Product
}
export interface avisRequest {
    note: number;
    commentaire: string;
    datePublication: string;
    utilisateurId: number | null;
    produitId: number | null ;
    
}

File: tempo-rise-api/src/main/java/tn/temporise/domain/model/Avis.java
package tn.temporise.domain.model;



import lombok.Builder;

import java.time.LocalDateTime;

@Builder

public record Avis (  Long id,
        int note,
        String commentaire,
        LocalDateTime datePublication,


        UtilisateurModel utilisateur,


        Produit produit){ }
